Include/cpython/dictobject.h

```diff
@@ -90,13 +90,18 @@ PyAPI_FUNC(PyObject *) _PyDictView_Intersect(PyObject* self, PyObject *other);
 
 /* Dictionary watchers */
 
+#define FOREACH_DICT_EVENT(V) \
```

Comment:
This (and the other ones) would need a PY_ prefix.

Reply:
Ah good call. I just copied the existing one for `FOREACH_FUNC_EVENT`, which I guess should have had the prefix but didn't.

I guess if we follow your other comment and move the `*_event_name` functions to the header and undef the macro, we don't need the prefix?

Reply:
`FOREACH_FUNC_EVENT` is also used in the unit tests currently (in `Modules/_testcapi/watchers.c`, which is why it already existed. So I guess I'll give it the prefix.

Reply:
I think even if we define then undefine the macro in a header, it could interfere with other user code that uses the same macro name, so we should still use the prefix.